### ISSUE TITLE
Yahoo contacts

### ISSUE THREAD

**Issue by mebibou** [MrSwitch/hello.js#431]
In the api list for Yahoo, you are not listing the `me/contacts` api, but when I look at the source code it seems you are actually loading the contacts through the `me/friends` api. Is this a mistake?

**Comment by MrSwitch** [comment]
Yeah it could be the definition between the two arose when some services had two different aproaches to contact management, i.e. Google has Gmail contacts and Google+ friends. I guess with Yahoo this is more like contacts. Is this a problem?

**Comment by mebibou** [comment]
Not a very big one but it means I can't have something generic to retrieve the contacts as in `hello(service).api('me/contacts')` where service is a variable.

**Comment by MrSwitch** [comment]
Sure, copy, paste and rename the me/friends definition in yahoo.js and create a PR. Thanks.
